src/store/zotac.ts

```diff
@@ -0,0 +1,14 @@
+import {Store} from './store';
+
+export const Zotac: Store = {
+	cartUrl: '',
+	links: [
+		{
+			brand: 'zotac',
+			model: 'trinity',
+			url: 'https://store.zotac.com/zotac-gaming-geforce-rtx-3080-trinity-zt-a30800d-10p',
```

Comment:
Zotac store is down and has a lot of catchpa right now. I'd suggest updating the `oosLabels`.